go.mod

```diff
@@ -12,12 +11,14 @@ require (
 	github.com/coreos/go-systemd/v22 v22.3.3-0.20220203105225-a9a7ef127534
 	github.com/docker/go-units v0.5.0
 	github.com/dolmen-go/contextio v0.0.0-20200217195037-68fc5150bcd5
+	github.com/elastic/beats/v7 v7.17.10
```

Comment:
is that right?

Reply:
No, this doesn't look right to me, thanks for catching.  No idea why it got added - same goes for some of the other changes in `go.mod`.  Let me try to revert these manually and see what happens.